### ISSUE TITLE
v6.3.0 introduces breaking change to webhook

### ISSUE THREAD

**Issue by msau42** [kubernetes-csi/external-snapshotter#927]
**What happened**:
The v6.3.0 introduced a dependency on the new alpha volumegroupsnapshots api, which causes the webhook to fail with:
```
volumegroupsnapshotclasses.groupsnapshot.storage.k8s.io is forbidden: User "system:snapshot-controller" cannot list resource "volumegroupsnapshotclasses" in API group "groupsnapshot.storage.k8s.io" at the cluster scope
```

**What you expected to happen**:
alpha features should be disabled by default

**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
- Driver version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by xing-yang** [comment]
Here's an PR to fix this: https://github.com/kubernetes-csi/external-snapshotter/pull/922

**Comment by sunnylovestiramisu** [comment]
@xing-yang Is there anyway to release a CVE fix without waiting for this breaking change to be fixed?

**Comment by msau42** [comment]
should be fixed in v6.3.1
